jep/1/README.adoc

```diff
@@ -575,11 +650,12 @@ contributors may request that the <<Reviewer>> for that JEP intervene.
 The reviewer will consider the matter, and render their decision,
 including describing what actions will be taken and documenting their reasoning.
 
-For disputes over JEP process or decisions made by a <<BDFL Delegate>>,
+If the <<Reviewer>> for a JEP is a <<BDFL Delegate>> or
```

Comment:
I'm not totally sure I understand this wording. A contributor may request that the BDFL intervene if the Reviewer is a BDFL Delegate? Why would they do that?